src/components/Workflow/SearchableWorkflowNameList.tsx

```diff
@@ -98,6 +167,105 @@ const padExecutionPaths = (items: WorkflowExecutionIdentifier[]) => {
+  isArchived ? <UnarchiveOutline /> : <ArchiveOutlined />;
+
+const SearchableWorkflowNameItemActions: React.FC<SearchableWorkflowNameItemActionsProps> =
+  React.memo(({ item, setHideItem }) => {
+    const styles = useStyles();
+    const { enqueueSnackbar } = useSnackbar();
+    const { id } = item;
+    const isArchived = isWorkflowArchived(item);
+    const [isUpdating, setIsUpdating] = useState<boolean>(false);
+    const [showConfirmation, setShowConfirmation] = useState<boolean>(false);
+
+    const mutation = useMutation(
+      (newState: WorkflowExecutionState) => updateWorkflowState(id, newState),
+      {
+        onMutate: () => setIsUpdating(true),
+        onSuccess: () => {
+          enqueueSnackbar(t('archiveSuccess', !isArchived), {
+            variant: 'success',
+          });
+          setHideItem(true);
+        },
+        onError: () => {
+          enqueueSnackbar(`${mutation.error ?? t('archiveError', !isArchived)}`, {
+            variant: 'error',
+          });
+        },
+        onSettled: () => {
+          setShowConfirmation(false);
+          setIsUpdating(false);
+        },
+      },
+    );
+
+    const onArchiveClick = (event: React.MouseEvent) => {
+      event.stopPropagation();
+      event.preventDefault();
+      setShowConfirmation(true);
+    };
+
+    const onConfirmArchiveClick = (event: React.MouseEvent) => {
+      event.stopPropagation();
+      event.preventDefault();
+      mutation.mutate(
+        isWorkflowArchived(item)
+          ? WorkflowExecutionState.NAMED_ENTITY_ACTIVE
+          : WorkflowExecutionState.NAMED_ENTITY_ARCHIVED,
+      );
+    };
+
+    const onCancelClick = (event: React.MouseEvent) => {
+      event.stopPropagation();
+      event.preventDefault();
+      setShowConfirmation(false);
+    };
+
+    return isUpdating ? (
+      <div className={classNames(styles.actionContainer, styles.centeredChild)}>
+        <CircularProgress size={24} />
+      </div>
+    ) : (
+      <div className={classNames(styles.actionContainer, showOnHoverClass)}>
+        {showConfirmation ? (
+          <div className={styles.confirmationBox}>
+            <Button
+              size="medium"
+              variant="contained"
+              color="primary"
+              className={styles.confirmationButton}
+              disableElevation
+              onClick={onConfirmArchiveClick}
+            >
+              {t('archiveAction', isArchived)}
+            </Button>
+            <Button
+              size="medium"
+              variant="contained"
+              color="inherit"
+              className={styles.confirmationButton}
+              disableElevation
+              onClick={onCancelClick}
+            >
+              {t('cancelAction')}
+            </Button>
+          </div>
+        ) : (
+          <IconButton
+            className={styles.centeredChild}
+            size="small"
+            title={t('archiveActionString', isArchived)}
```

Comment:
`archiveActionString` ln 259 doesn't exist in the corresponding `strings.ts`, it's `archiveAction` as used on ln 242, can you please update it? There's also one other occurrence with `archiveActionString` in `cells.tsx` https://github.com/flyteorg/flyteconsole/blob/master/src/components/Executions/Tables/WorkflowExecutionTable/cells.tsx#L118 could you also amend this one? 🙏